bluemira/utilities/tools.py

```diff
@@ -421,6 +421,15 @@ def __set__(self, obj: Any, value: str | tuple[float, ...] | ColorPalette):
         setattr(obj, self._name, value)
 
 
+def iterable_to_list(obj: Any | Iterable[Any]):
```

Comment:
Minor: does this cause downstream type issues? If so, making it generic would be better.